app/src/main/res/drawable/ic_onboarding_tracking_protection.xml

```diff
@@ -2,23 +2,29 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:aapt="http://schemas.android.com/aapt"
-        android:width="24dp"
-        android:height="24dp"
-        android:viewportWidth="24"
-        android:viewportHeight="24">
+<vector
```

Comment:
Is this missing?

`<?xml version="1.0" encoding="utf-8"?>`

Reply:
No,  I just updated the vector part here you can see the raw file https://github.com/mozilla-mobile/fenix/blob/f0bc8bbd87f886671ee0d5b86dd679d4bc0f2fcb/app/src/main/res/drawable/ic_onboarding_tracking_protection.xml